### ISSUE TITLE
[Panel] Buttons in tool bar the right corner are invisible in Storybook's Isolation Mode.

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3047]
## Current behavior

Using Storybook's Isolation Mode to view Panel stories that have buttons inside tool bar makes this buttons' icons invisible. 
Mouse user still can hover over places in which buttons should be, and there is change in button background. 
Keyboard-only users can place their focus on buttons and focus indicator is visible, but button's icon is still hidden. 
`sl-icon` is present in HTML code inside button.

<img width="1100" height="113" alt="Image" src="https://github.com/user-attachments/assets/29f99787-f086-4b2d-affd-353ad466724b" />
<img width="377" height="116" alt="Image" src="https://github.com/user-attachments/assets/8a6ba038-fa21-4bd9-b0d9-1d1f66b914d0" />
<img width="1900" height="81" alt="Image" src="https://github.com/user-attachments/assets/e5612429-432d-411e-ab60-cf96dac1f43b" />

<img width="1898" height="352" alt="Image" src="https://github.com/user-attachments/assets/f3fb9e3c-4222-4779-a208-e7ed3808d581" />

https://github.com/user-attachments/assets/c0634cd7-734d-49c0-a5ef-f55086fbae9e

## Suggested solution

CSS for that icons should be investigated to find what is blocking display of each button's icon. 

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
